indico/modules/events/sessions/controllers/management/__init__.py

```diff
@@ -0,0 +1,43 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask import request
+
+from MaKaC.webinterface.rh.conferenceModif import RHConferenceModifBase
+
+
+class RHManageSessionsBase(RHConferenceModifBase):
+    """Base RH"""
+
+    def _checkParams(self, params):
+        RHConferenceModifBase._checkParams(self, params)
+        self.event = self._conf.as_event
```

Comment:
There's `event_new` that contains the `Event` and using `event` for this just adds a MASSIVE amount of confusion (older RHs use `self.event` for the legacy conference)